### DISCUSSION TITLE
How to package module

### DISCUSSION THREAD

**Discussion by hyzx86** [OrchardCMS/OrchardCore#13170] [Q&A] [answered]
The module cannot be found with a packaged reference, but it can be found with a direct reference from the project

https://github.com/EasyOC/EasyOC.Modules/tree/main/src/EasyOC.ReplaceAction

Did I miss a step?

**Comment by emrahtokalak** [comment, marked as answer]
You should make the necessary changes for package generation in your csproj file. (https://github.com/EasyOC/EasyOC.Modules/blob/main/src/EasyOC.ReplaceAction/EasyOC.ReplaceAction.csproj)

Related documentation: https://learn.microsoft.com/en-us/nuget/create-packages/creating-a-package-msbuild#automatically-generate-package-on-build

Example csproj file: https://github.com/emrahtokalak/Emrah.OrchardCore.CKEditor/blob/main/src/Emrah.OrchardCore.CKEditor/Emrah.OrchardCore.CKEditor.csproj

**Reply by hyzx86** [reply]
Thanks 。。。 but still not work.

This problem seems to only occur in my solution. 
I tried to reference this package in the OrchardCore source code solution, and it can be normally shown in the feature list.

I need to find the problem in my solution

**Reply by hyzx86** [reply]
Done ， remove some extra properties

**Reply by hyzx86** [reply]
It must be referenced as such, or directly in the startup project

```xml
<PackageReference Include="EasyOC.ReplaceAction" PrivateAssets="none"/>

```

**Reply by Piedone** [reply]
And you can use this to auto-publish it with GitHub Actions: https://github.com/Lombiq/GitHub-Actions/blob/dev/Docs/Workflows.md#nuget-publish-workflow

**Reply by hyzx86** [reply]
@Piedone Thanks,That was my first nuget package and I was looking for a similar script, thank you very much 👍
